pkg/services/sqlstore/user.go

```diff
@@ -317,7 +317,7 @@ func GetUserOrgList(query *models.GetUserOrgListQuery) error {
 	sess.Join("INNER", "org", "org_user.org_id=org.id")
 	sess.Where("org_user.user_id=?", query.UserId)
 	sess.Cols("org.name", "org_user.role", "org_user.org_id")
-	sess.OrderBy("org.name")
+	sess.OrderBy("LOWER(org.name),org.name")
```

Comment:
Can you please revert this change? After discussing the matter internally, we've decided that it has too much of a performance impact to lowercase org.name as part of the query.